src/nvim/strings.c

```diff
@@ -413,21 +413,18 @@ char_u *vim_strchr(const char_u *string, int c)
 }
 
 /*
- * Version of strchr() that only works for bytes and handles unsigned char
- * strings with characters above 128 correctly. It also doesn't return a
- * pointer to the NUL at the end of the string.
+ * Search for c in buf. Appropriately handles values of c in [0, 255].
```

Comment:
Please use Doxygen style comments `///`, and function parameters should be surrounded in backticks, e.g. ``` `c` ```.